<launch>

    <node pkg="traj_view" type="roomba_controller.py" name="trajController20" >
        <rosparam param="name">"roomba20"</rosparam>
        <rosparam param="action_topic">"/move_base_simple/goal20"</rosparam>
        <rosparam param="init_pos">[0, 0, 0]</rosparam>
    </node>

    <node pkg="traj_view" type="roomba_controller.py" name="trajController21" >
        <rosparam param="name">"roomba21"</rosparam>
        <rosparam param="action_topic">"/move_base_simple/goal21"</rosparam>
        <rosparam param="init_pos">[0, -2, 0]</rosparam>
    </node>
    <node pkg="map_server" type="map_server" args="/home/redwan/catkin_ws/src/traj_view/config/mymap.yaml" name="mapserver" />
    <node pkg="rviz" type="rviz" name="roomba_rviz" args="-d /home/redwan/catkin_ws/src/traj_view/config/multi_agent_traz_top.rviz" />

    <node pkg="traj_view" type="multi_agent_cntrl_node.py" name="trajViewer" output="screen">
        <rosparam param="radius">0.4</rosparam>
        <rosparam param="robot20">[[0, 0], [1.5, 0], [1.5, -1], [0, -1], [0, -2], [0, -2]]</rosparam>
        <rosparam param="robot21">[[0, -2], [1.5, -2], [1.5, -1], [0, -1], [0, 0], [0, 0]]</rosparam>
        <rosparam param="goal_topics">['/move_base_simple/goal20', '/move_base_simple/goal21']</rosparam>
    </node>

</launch>